integration/tests/identity_provider/identity_provider_test.go

```diff
@@ -0,0 +1,379 @@
+//go:build integration
+// +build integration
+
+//revive:disable
+package identity_provider
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
```

Comment:
Please order the imports. :)

Reply:
isn't this ordered already?

Reply:
technically, each segment is, yes. I meant to pull them together instead of having them haphazardly all over the place. 

We started to apply the rule of the following order:

- stdlib
- third party
- organisation
- package local

To separate the imports into clean segments.

Reply:
@Skarlso, which imports are not in the order you stated?

Reply:
Never mind. :)))) It's fine. :)